config/README.md

```diff
@@ -153,6 +153,15 @@ sparse_mode = false
 build_info = true
 build_info_path = "build-info"
 root = "root"
+# Configres permissions for cheatcodes that touch the filesystem like `vm.writeFile`
+# `permission` restricts the type of access, possible values are
+#    `true` | "read-write" => `read` + `write` access allowed (`vm.readFile` + `vm.writeFile`)
+#    `false` | "none" => no access
+#    `read` => only read access (`vm.readFile`)
+#    `write` => only write access (`vm.writeFile`)
+# The `allowed_paths` further lists the paths that are considered, e.g. `./` represents the project root directory
+# By default _no_ fs access permission is granted, and _no_ paths are allowed
+fs_permissions = { permission = false, allowed_paths = [] }
```

Comment:
how you considered per path control? so getting read-write wouldn't apply on all the paths?

Reply:
`Configres` typo

Reply:
that makes sense, I thought of permission as a toggle for all paths but I guess it makes more sense to use this on a per path bases

Reply:
So I think I'll make `fs_permission` an array/table of `{ permission = "none|read|write|read-write", path = "" }`